### ISSUE TITLE
如何在 PaddleSeg 或者 HumanSeg 中复现 paddlehub 中的 deeplabv3p_xception65_humanseg 的效果

### ISSUE THREAD

**Issue by yangshiyu89** [PaddlePaddle/PaddleSeg#663]
您好，

最近在研究 humanseg 相关技术。尝试了一下 paddlehub 中的 deeplabv3p_xception65_humanseg 模型效果很好。不过想在 PaddleSeg 中或者在 contrib/HumanSeg 中进行复现，提取更多的信息学习一下。我的步骤大概如下：

1. 将 paddlehub 下 deeplabv3p_xception65_humanseg 模型通过 module.save_inference_model(dirname), 将模型保存为 ``__model__`` 和 ``__params__`` 两个文件放在指定文件夹下，以适配 HumanSeg 的文件要求，然后将文件夹复制在 contrib/HumanSeg/pretrained_weights/ 中。

2. 以 humanseg_sever_inference 中的 model.yml 文件作为模板，修改 target_size 为 513, 513，mean 为 104.008，116.669， 122.675, std 为 1，1，1 (这些是根据 ~/.paddlehub/modules/deeplabv3p_xception65_humanseg/data_feed.py 中的配置进行修改的)，并且修改 修改 transforms/functional.py 中 def normalize, 去掉 / 255.0

3. 修改 infer.py 函数中 ``--image_shape`` 默认大小为 [513, 513]。

但是，到这一步，就会报 result[1] 越界的错误。
`Exception has occurred: IndexError
list index out of range
  File "~/PaddleSeg/contrib/HumanSeg/models/humanseg.py", line 710, in predict
    logit = result[1]`

之后虽然经过一些操作，虽然将程序跑通，但是并不能复现在paddlehub下的效果，分割效果很渣。并且这些修改动了很多 HumanSeg 的源码，并且对于运行其他官方的预训练模型也不能复用，这肯定不是对的方式。想请问相关人员，如何在 PaddleSeg 或者 HumanSeg 中，准确无误地复现 paddlehub 中的 deeplabv3p_xception65_humanseg 模型？

**Comment by wuyefeilin** [comment]
hub中deeplabv3p_xception65_humanseg 和humanseg中的模型略有差异，主要是数据预处理和模型最后的输出， 你可以用vdl打开两者的模型结构对比一下看看。 均值需要除以255， std为0.5

**Comment by yangshiyu89** [comment]
> hub中deeplabv3p_xception65_humanseg 和humanseg中的模型略有差异，主要是数据预处理和模型最后的输出， 你可以用vdl打开两者的模型结构对比一下看看。 均值需要除以255， std为0.5

如您所述，对于输入输出，hub 中和 humanseg 最大的不同在于输入的预处理。如我上述，我在 yml 文件和 function.py 文件分别做了预处理的修改适配 hub 中的模型。可是调整过后，结果依然不理想。
